src/pages/signup-affiliates/components/_account-personal-details.tsx

```diff
@@ -0,0 +1,333 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { WizardStepProps } from '../_types'
+import affiliate_validation from '../validations/_affilaite_validation'
+import BirthForm from '../utils/_birth-form'
+import AffiliateInput from '../utils/_affiliate-input'
+import { localize } from 'components/localization'
+import device from 'themes/device'
+
+const InputGroup = styled.div`
+    display: flex;
+    flex-direction: row;
+    inline-size: 100%;
+    block-size: 100%;
+    margin-bottom: 4rem;
+    justify-content: center;
+
+    p {
+        min-height: 30px !important;
+    }
+`
+
+const InputWrapper = styled.div`
+    width: 60%;
+    line-height: 10px;
+
+    @media ${device.mobileL} {
+        width: 250px;
+    }
+`
+
+const PersonalDetails = ({
+    is_individual,
+    affiliate_data,
+    updateData,
+    onValidate,
+}: WizardStepProps) => {
+    const [username, setUsername] = useState(affiliate_data.username)
+    const [first_name, setFirstName] = useState(affiliate_data.first_name)
+    const [last_name, setLastName] = useState(affiliate_data.last_name)
+    const [date_birth, setDateBirth] = useState(affiliate_data.date_birth)
+    const [company_name, setCompanyName] = useState(affiliate_data.company_name)
+    const [company_registration_number, setCompanyRegistrationNumber] = useState(
+        affiliate_data.company_registration_number,
+    )
+    const [website_url, setWebsiteUrl] = useState(affiliate_data.website_url)
+    const [social_media_url, setSocialMedia] = useState(affiliate_data.social_media_url)
+    const [password, setPassword] = useState(affiliate_data.password)
```

Comment:
There are a lot of TS errors, seems like ts can’t refer to the correct type , I think in `_types.ts` for the `AffiliateAccountTypes` need to make the objects inside as a different type it self so ts can refer to it,  correct me If I'm wrong thanks 🙏 

![Screenshot 2023-09-27 at 10 53 52 AM](https://github.com/binary-com/deriv-com/assets/64970259/656762a9-6b3b-44c9-8df5-1556df95d8e5)

Reply:
Yeah, I worked on it and resolve already. please check again
Personal thanks that you fetched my pr :)